include/constants/flags.h

```diff
@@ -1397,8 +1397,8 @@
 #define FLAG_SYS_SAFARI_MODE                        (SYSTEM_FLAGS + 0x2C)
 #define FLAG_SYS_CRUISE_MODE                        (SYSTEM_FLAGS + 0x2D)
 
-#define FLAG_UNUSED_0x88E                           (SYSTEM_FLAGS + 0x2E) // Unused Flag
-#define FLAG_UNUSED_0x88F                           (SYSTEM_FLAGS + 0x2F) // Unused Flag
+#define FLAG_GOT_VS_SEEKER                          (SYSTEM_FLAGS + 0x2E)
+#define FLAG_SYS_VS_SEEKER_CHARGING                 (SYSTEM_FLAGS + 0x2F)
```

Comment:
tabs vs. spaces

these flags should probably be configurable ( ? )

Reply:
We actually don't need `FLAG_GOT_VS_SEEKER` at all, so I'll be removing it, but without `FLAG_SYS_VS_SEEKER_CHARGING`, the Vs. Seeker will be in a weird state where the Vs. Seeker APPEARS to be working, but its not.

So we could
1. Modify the item so that it ONLY works when this flag is set. Should just be a one line change.
2. Make this flag permanent? I actually don't know what the expansion philosophy is around this.